test/extended/networking/multicast.go

```diff
@@ -167,7 +166,7 @@ func launchTestMulticastPod(f *e2e.Framework, nodeName string, podName string) e
 			Containers: []kapiv1.Container{
 				{
 					Name:    contName,
-					Image:   "openshift/test-multicast",
+					Image:   "quay.io/openshift/community-e2e-images:e2e-docker-io-openshift-test-multicast-latest-4AxcBBxKg_prX34z",
```

Comment:
That URL seems pretty sketchy. Is that actually stable?

Reply:
it's what we have in [4.7](https://github.com/openshift/origin/blob/master/test/extended/util/image/zz_generated.txt#L12). I couldn't find a better way to indirect other than using the whole `image.LocationFor` API which doesn't exist today in 4.6 at all.